Controller/ChangePasswordFOSUser1Controller.php

```diff
@@ -33,7 +33,7 @@ public function changePasswordAction()
     {
         $user = $this->getUser();
         if (!is_object($user) || !$user instanceof UserInterface) {
-            $this->createAccessDeniedException('This user does not have access to this section.');
+            throw $this->createAccessDeniedException('This user does not have access to this section.');
```

Comment:
This change is correct, but there is a problem... this method we are calling is not present on SF < 2.5:

http://symfony.com/blog/new-in-symfony-2-5-create-the-correct-denied-http-exception

IMHO we could simply replace this call by:

`throw new AccessDeniedException('This user does not have access to this section.')`

but there could be other ways like implement this method as a fallback for SF 2.3:

https://github.com/symfony/symfony/blob/2.5/src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php#L166

Or do an `if` to check if the method exists.